app/components/SessionTile.jsx

```diff
@@ -0,0 +1,59 @@
+import { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Component from '../Component';
+import style from './SessionTile.scss';
+import ClassNames from 'classnames';
+import translate from '../i18n/Translate';
+import { Card, CardTitle, CardActions } from 'react-toolbox/lib/card';
+import { default as Button } from 'react-toolbox/lib/button';
+
+const stateToProps = state => ({ });
+
+const actionsToProps = dispatch => ({});
+
+@translate('SessionTile')
+@translate('SessionName')
+@connect(stateToProps, actionsToProps)
+class SessionTile extends Component {
+    renderLastJoined (lastJoined) {
+        const MINUTE = 60 * 1000;
```

Comment:
You could use momentjs if you wanted to simplify this: http://momentjs.com/docs/#/displaying/fromnow/